transformations/city_names_transformation/requirements.txt

```diff
@@ -0,0 +1 @@
+es_core_news_sm @ https://github.com/explosion/spacy-models/releases/download/es_core_news_sm-3.0.0/es_core_news_sm-3.0.0.tar.gz
```

Comment:
Please add this to a separate file called [requirements-disabled.txt](https://github.com/GEM-benchmark/NL-Augmenter/blob/fd9b699fc75824db942bdf485f49dc5c31bf60dc/transformations/pinyin/requirements-disabled.txt)